src/assets/example_document.jsx

```diff
@@ -0,0 +1,57 @@
+import { getMarkForCommentThreadID } from "../utils/editorCommentUtils";
+import { v4 as uuid } from "uuid";
+
+const overlappingCommentThreadID = uuid();
+
+// THERE ARE 4 types of redactions: suggested, rejected, accepted, current 
```

Comment:
I think we should pull these names out into constants instead of hardcoding them in eventually